testing/falkon/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Contributor: Leo <thinkabit.ukim@gmail.com>
+# Maintainer: Leo <thinkabit.ukim@gmail.com>
+pkgname=falkon
+pkgver=3.1.0
+pkgrel=0
+pkgdesc="Cross-platform Qt Web Browser"
+url="https://github.com/KDE/falkon"
+arch="all"
+license="GPL-3.0-or-later"
+depends="qt5-qtbase-sqlite"
+makedepends="cmake python3 qt5-qtbase-dev extra-cmake-modules qt5-qtdeclarative-dev
+ qt5-qtwebengine-dev xcb-util-dev libxcb-dev qt5-qtx11extras-dev qt5-qttools-dev
+ libgnome-keyring-dev"
```

Comment:
Why is a GNOME dependency here? It's a KDE browser, it shouldn't use anything GNOME at all. Also, support for gnome-keyring has been removed https://github.com/KDE/falkon/commit/b9fb89a31ca6966412a60a30f4080949839aa839. This isn't in a release yet but will be in the next release, so I wouldn't encourage users from using the functionality now.

Reply:
fixed